config/connection.js

```diff
@@ -17,5 +17,17 @@ if(process.env.JAWSDB_URL){
         }
     );
 }
-
+if(process.env.CLOUDINARY_URL){
```

Comment:
make a new object like

const cloudinary = require('cloudinary').v2; at the top

then add the config:
cloudinary.config({
  secure: true
});

change last line to:
module.exports = {sequelize, cloudinary};